src/cascadia/TerminalApp/ActionAndArgs.cpp

```diff
@@ -284,6 +288,7 @@ namespace winrt::TerminalApp::implementation
                 { ShortcutAction::ToggleFocusMode, RS_(L"ToggleFocusModeCommandKey") },
                 { ShortcutAction::ToggleFullscreen, RS_(L"ToggleFullscreenCommandKey") },
                 { ShortcutAction::ToggleAlwaysOnTop, RS_(L"ToggleAlwaysOnTopCommandKey") },
+                { ShortcutAction::SendInput, RS_(L"SendInputCommandKey") },
```

Comment:
IMO this should be mapped to `L""`, so that we don't accidentally generate a name for a `sendInput` action that doesn't have a value for `input`

see for example: `CloseOtherTabs`